bin/ds-global

```diff
@@ -61,6 +61,7 @@ function enable {
 
   # Fix Campaign translatable title values.
   drush ds-global-titles-fix campaign
+  drush ds-global-titles-fix campaign_run
```

Comment:
so stick that drush addition I made in a new function like `make_more_campaign_fields_translatable`, and then add it to the else if block below so you can specify it